inst/examples/app/analytics/app.R

```diff
@@ -30,6 +30,13 @@ if (Sys.getenv("R_CONFIG_ACTIVE") == "rsconnect") {
   )
 }
 
-analytics_app(data_storage = data_storage)
+# check if there is data in the data_storage
+if (nrow(data_storage$read_event_data()) > 0) {
+  # run the analytics app if there is data
+  analytics_app(data_storage = data_storage)
+} else {
+  # the empty app if there is no data
+  empty_app()
+}
 
 # shiny::shinyAppDir(file.path("inst", "examples", "app", "analytics"))
```

Comment:
Weird that this only pops up now. (note that this comment is a trick to fool RStudio to think this is a Shiny App)

```suggestion
# shiny::shinyAppDir(file.path("inst", "examples", "app", "analytics")) # nolint: commented_code.
```

Reply:
I was aware of the lint error.
But since it was already in the main branch and without it RStudio couldn't recognize the file as shiny app, I thought it was there on purpose.
I'll fix this in next commit.